app/views/health/backup_plans/_backup_plan_list.haml

```diff
@@ -1,4 +1,4 @@
-.jBackupPlansList.overflow-scroll
```

Comment:
historically, these have been necessary because the list can be pretty wide depending on what's in it, and user's don't always have a screen big enough to see it.  I'm not seeing this moved up into the CSS, did something else change to make it unnecessary?

Reply:
Oh good catch. I will change it to `overflow-auto`. The problem with overflow-scroll is that it leaves extra padding for scrollbars even when you don't need them:
![Screen Shot 2022-09-28 at 12 07 55 PM](https://user-images.githubusercontent.com/5333982/192830498-03007859-dc61-4f81-98c4-8ad4055046ac.png)


With `overflow: auto`, the padding only appears when you actually have scroll bars. I think I may have removed it in a few other places, so I will check.